### PR TITLE
orderbook-adjustments

### PR DESCRIPTION
**Pull request by JohnDanz**
https://github.com/AugurProject/augur/issues/5020
adjusted labels and alignment of the order book

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://dashboard.featurepeek.com). ⚡️
Your deployment will be kept up-to-date with this pull request's latest changes.

**https://peek.run/w9jhxw1**

-------

*Please read [our docs](https://docs.featurepeek.com/configuring-peek-yml/) for more configuration details.*

File: packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx [hunk 1]
```diff
@@ -204,7 +204,7 @@ export default class OrderBook extends Component<
   render() {
     const { pricePrecision, toggle, hide, marketType, hasOrders, orderBook } = this.props;
     const {hoveredSide, hoveredOrderIndex} = this.state;
-
+    console.log(orderBook.spread);
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market-charts/components/order-book/order-book.tsx]: did you mean to keep this?

File: packages/augur-ui/src/modules/market-charts/components/order-book/order-book.styles.less [hunk 2]
```diff
@@ -84,13 +83,14 @@
...
         grid-column: 1 / span 1;
         color: @color-primary-text;
-        margin-left: auto;
+        margin-left: @size-40;
       }
 
       &:nth-of-type(2) {
         grid-column: 2 / span 1;
-        padding-right: @size-16;
+        // padding-right: @size-16;
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market-charts/components/order-book/order-book.styles.less]: keep?

**Review by phoebemirman — CHANGES_REQUESTED** [review]
left comment

**Comment by JohnDanz** [comment]
https://github.com/AugurProject/augur/issues/5050

https://github.com/AugurProject/augur/issues/5020

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**